libs/facilities/shared/models/README.md

```diff
@@ -0,0 +1,3 @@
+# facilities-shared-models
+
+This library contains shared models
```

Comment:
Pls enhance this. It is a library which contains models like interfaces, types or classes that are used throughout the frontend and backend of the facilities domain

Reply:
I dont even know what this lib does in there, i just removed it